.github/workflows/ci.yml

```
@@ -0,0 +1,14 @@
+name: Continuous Integration
+on: [pull_request]
```

I didn't try GitHub Actions myself, but it seems like we could also use both (just to be sure)

```suggestion
on: [push, pull_request]
```

https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-using-a-list-of-events

because I think that the `master` (or `main`) branch won't be rebuild on merge if we just use `pull_request`, but again, I never tried, so => 🤷

---

Discussed. We'll make this evolve later if needed.